generators/app/templates/src/redux/authHandlers.js

```diff
@@ -106,10 +106,10 @@ export function reducer(state = Immutable(defaultState), action) {
 
 export function propTypes() {
```

Comment:
Additional comments for this file:
Put in `actionCreators`only these being exported.
I'd move `PropTypes` to `reducers`
Extract route names
Add namespaces
Put this `actions.js` into `redux/auth`